tests/osaka/eip7692_eof_v1/eip7480_data_section/test_code_validation.py

```diff
@@ -143,14 +202,11 @@ def container_name(c: Container):
     VALID,
     ids=container_name,
 )
-def test_legacy_initcode_valid_eof_v1_contract(
+def test_valid_containers_with_data_section(
```

Comment:
I changed the names and docstrings of these 2 tests because they didn't make any sense to me.